### PR TITLE
186963497 Update Outcome Styling

### PR DESCRIPTION
**Pull request by tealefristoe**
PT Story: https://www.pivotaltracker.com/story/show/186963497

This PR makes one major and a couple of minor styling changes to the output messages.
- The major change is adding an svg border with styling that matches the experiment's graph curve style. This involved moving some css from a specific file to a more general file.
- Added "Result: " before each output message.
- Added a subtle box shadow to the output message area.

### CODE REVIEW

File: src/components/new-simulation/simulation-outcome.tsx [hunk 1]
```diff
@@ -28,9 +30,35 @@ export function SimulationOutcome({ control1, control2, outcomeHidden, setOutcom
...
+    const borderWidth = 392; // Based on .outcome-area width in simulation-outcome.scss - margin space
+    const borderRadius = 3;
+    const bWidth = borderWidth - (2 * borderRadius + strokeWidth);
+    const bHeight = borderHeight - (2 * borderRadius + strokeWidth);
+    const aPrefix = `${borderRadius},${borderRadius} 0 0 1`;
+    const top = `h ${bWidth} a ${aPrefix} ${borderRadius},${borderRadius}`;
+    const right = `v ${bHeight} a ${aPrefix} -${borderRadius},${borderRadius}`;
+    const bottom = `h -${bWidth} a ${aPrefix} -${borderRadius},-${borderRadius}`;
+    const left = `v -${bHeight} a ${aPrefix} ${borderRadius},-${borderRadius}`;
+    const path = `M ${borderRadius + strokeWidth / 2},${strokeWidth / 2} ${top} ${right} ${bottom} ${left} z`;
```
Comment (bgoldowsky) [inline src/components/new-simulation/simulation-outcome.tsx]: OK, you don't want me to point this out _after_ you've done all this work, but `<rect>` in SVG supports `rx` and `ry` for rounding corners simply.  [docs](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/rect)

**Review by bgoldowsky — APPROVED** [review]
Looks good - one suggestion for simplification.

File: src/components/new-simulation/simulation-outcome.tsx (hunk 1 above)
Comment (tealefristoe) [reply src/components/new-simulation/simulation-outcome.tsx]: 😱 ... 🤦 
Wow. Thank you for pointing that out.

**Review by bgoldowsky — APPROVED** [review]
Looks great!

**Pull request merged**